sky/skylet/job_lib.py

```diff
@@ -884,7 +884,7 @@ def tail_logs(cls,
                   follow: bool = True) -> str:
         # pylint: disable=line-too-long
         code = [
-            f'job_id = {job_id} if {job_id} is not None else job_lib.get_latest_job_id()',
+            f'job_id = {job_id} if {job_id} != None else job_lib.get_latest_job_id()',
```

Comment:
Why change this? IIUC it is recommended in the google style guide that use `is/is not` to judge whether the variable is None or not?

Reply:
`{job_id}` here is a literal instead of a variable in the generated code. If `job_id == 1` the generated code will produce the following warning if we use `is not`:
```
>>> 1 is not None
<stdin>:1: SyntaxWarning: "is not" with a literal. Did you mean "!="?
```